source/scripts/model/Monster.js

```diff
@@ -110,4 +118,11 @@ export default class Monster {
             this.game.remove("monsters", this)
         }
     }
+    moveOnScreen() {
```

Comment:
Since this is a getter, can we name it something like getFirstMovement getOffscreenMovement or something like that?